jobs/gtfs-rt-parser-v2/gtfs_rt_parser.py

```diff
@@ -492,14 +500,32 @@ def parse_and_upload(
                     )
                 outcomes.append(
                     RTFileProcessingOutcome(
-                        step="parse",
+                        step=RTProcessingStep.parse,
                         success=False,
                         exception=ValueError(msg),
                         extract=extract,
                     )
                 )
                 continue
 
+            if "entity" not in parsed:
+                msg = f"WARNING: no parsed entity found in {str(extract.path)}"
+                if verbose:
+                    log(
+                        msg,
+                        fg=typer.colors.YELLOW,
+                        pbar=pbar,
+                    )
+                outcomes.append(
+                    RTFileProcessingOutcome(
+                        step=RTProcessingStep.parse,
+                        success=True,
+                        extract=extract,
+                        header=parsed["header"],
```

Comment:
Not to overcomplicate, but genuinely not sure: Do we need to distinguish between a case where there is no content at all vs. header and no entities? Like, is it possible to have an entirely empty proto? Basically, wondering if it is possible for us to get a `KeyError` here.

Reply:
Ok